src/components/helper-methods.js

```diff
@@ -58,8 +59,10 @@ export default {
 
   getCalculatedValues(props, defaultStyles) {
     const style = Helpers.getStyles(props.style, defaultStyles, "auto", "100%");
-    const colors = Array.isArray(props.colorScale) ?
-      props.colorScale : Style.getColorScale(props.colorScale);
+    const colorScale = props.theme && props.theme.pie ? props.theme.pie.colorScale
+    : props.colorScale;
```

Comment:
actually `props.colorScale` should take precedence over the color scale in theme. You will need to remove `colorScale` from `defaultProps` though and provide the fallback value in this calculation.